svg/interact/script-content.svg

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     xmlns:h="http://www.w3.org/1999/xhtml">
+  <h:link rel="help" href="https://svgwg.org/svg2-draft/interact.html#ScriptElement"/>
+  <h:meta name="assert" content="The script element is in the content model of all elements."/>
+  <h:script src="/resources/testharness.js"/>
+  <h:script src="/resources/testharnessreport.js"/>
+
+<script>function s0() { return s1(); }</script>
+
+<a>
+  <script>function s1() { return s2(); }</script>
+</a>
+
+<g>
+  <script>function s2() { return s3(); }</script>
+</g>
+
+<animate>
+  <script>function s3() { return s4(); }</script>
+</animate>
+
+<animateMotion>
+  <script>function s4() { return s5(); }</script>
+  <mPath>
+    <script>function s5() { return s6(); }</script>
+  </mPath>
+</animateMotion>
+
+<animateTransform>
+  <script>function s6() { return s7(); }</script>
+</animateTransform>
+
+<circle>
+  <script>function s7() { return s8(); }</script>
+  <clipPath>
+    <script>function s8() { return s9(); }</script>
+  </clipPath>
+</circle>
+
+<defs>
+  <script>function s9() { return s10(); }</script>
+</defs>
+
+<desc>
+  <script>function s10() { return s11(); }</script>
+</desc>
+
+<discard>
```

Comment:
The `<discard>` element is new in SVG Animations Level 2; I don't think it's supported anywhere.

But I don't think that should affect the function of a script element that it wraps.  Inside a .svg file, unknown elements won't change the parsing context.